docs/src/main/tut/typeclasses/contravariantmonoidal.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: docs
+title: "Contravariant Monoidal"
+section: "typeclasses"
+source: "core/src/main/scala/cats/ContravariantMonoidal.scala"
+scaladoc: "#cats.ContravariantMonoidal"
+---
+# Contravariant Monoidal
+
+The `ContravariantMonoidal` type class is for [`Contravariant`](contravariant.html) functors that can define a
+`product` function and a `unit` function.
+
+```tut:book:silent
+import cats.Contravariant
+
+trait ContravariantMonoidal[F[_]] extends Contravariant[F] {
```

Comment:
Maybe we shouldn't use `tut` here as it shadows the real `ContravariantMonoidal`.

Reply:
we could still use `tut` here then add a `tut:reset` after ([explained here](https://github.com/tpolecat/tut#modifiers)) so that it has no effect after.